treeherder/webapp/api/serializers.py

```diff
@@ -185,19 +185,7 @@ def to_representation(self, failure_line):
 class TextLogErrorSerializer(serializers.ModelSerializer):
     class Meta:
         model = models.TextLogError
-        exclude = ["step"]
-
-
-class TextLogStepSerializer(serializers.ModelSerializer):
-    errors = TextLogErrorSerializer(many=True, read_only=True)
-    result = serializers.SerializerMethodField()
-
-    def get_result(self, obj):
-        return obj.get_result_display()
-
-    class Meta:
-        model = models.TextLogStep
-        exclude = ["job"]
+        exclude = []
```

Comment:
The mentioned intention is to use `fields` instead.

Reply:
odd, I must have screwed something up, let me get this fixed to work with the comment I added indicating I used `fields = 'all'`